.stylelintrc.js

```diff
@@ -12,42 +11,24 @@ module.exports = {
     '**/docs/public/**',
     '**/docs/static/**',
   ],
-  plugins: ['stylelint-scss', 'stylelint-order'],
+  plugins: ['stylelint-order'],
   reportDescriptionlessDisables: true,
-  reportNeedlessDisables: true,
   reportInvalidScopeDisables: true,
+  reportNeedlessDisables: true,
   rules: {
-    // Turning this off in favor of 'at-rule-allowed-list'
-    'at-rule-no-unknown': null,
-    'at-rule-allowed-list': [
-      [
-        'function',
-        'if',
-        'use',
-        'media',
-        'supports',
-        'font-face',
-        'keyframes',
-        'extend',
-        'mixin',
-        'include',
-        // TODO: At some point we will be switching from 'import' to 'use'
-        'import',
-        // SCSS at-rules
-        'forward',
-        'container',
-        'each',
-      ],
-      { severity: 'warning' },
-    ],
-    // Importants should only be used in our utility classes if possible
+    'comment-empty-line-before': null,
+    'custom-property-pattern': null,
+    // `!important` should only be used in our utility classes if possible
     'declaration-no-important': [true, { severity: 'warning' }],
-    'function-no-unknown': [true, { ignoreFunctions: ['percentage', 'math.div'] }],
+    'keyframes-name-pattern': null,
     'no-descending-specificity': null,
-    'no-invalid-position-at-import-rule': null,
+    'selector-class-pattern': null,
+    'value-keyword-case': ['lower', { ignoreKeywords: ['currentColor', 'optimizeSpeed'] }],
     'order/order': ['custom-properties', 'declarations', 'rules'],
     'order/properties-alphabetical-order': true,
-    'scss/dollar-variable-no-missing-interpolation': true,
-    'scss/selector-no-redundant-nesting-selector': true,
+    'scss/at-extend-no-missing-placeholder': null,
+    'scss/at-mixin-pattern': null,
```

Comment:
https://github.com/stylelint-scss/stylelint-scss/blob/master/src/rules/at-mixin-pattern/README.md

Reply:
Defaulted to wanting all mixins written in kebab case, but we don't really have a standard for how those are written.